docs/src/description/constraints.md

```diff
@@ -4,7 +4,7 @@
 
 The `boundary_constraints` section consists of expressions describing the expected value of columns in the main or auxiliary traces at the specified boundary. Column boundaries can be selected using boundary accessors. Valid boundary accessors are `.first`, which selects the first cell of the column to which it is applied, and `.last`, which selects the last cell of the column column to which it is applied.
 
-**Boundary constraints are required.** The `boundary_constraints` section must be defined and contain at least one boundary constraint.
+**Boundary constraints are required.** The `boundary_constraints` section must be defined and should contain at least one boundary constraint.
```

Comment:
nit: non-blocking, but I think this was more correct before, because we do require that the section "must contain" at least one boundary constraint.